docs/content/preview/explore/fault-tolerance/macos.md

```diff
@@ -79,15 +47,15 @@ Next, create a 3 node cluster by joining two more nodes with the previous node.
                 --join=127.0.0.1
 ```
 
-## 2. Run the sample key-value app
+## Run the sample key-value app
 
-Download the YugabyteDB workload generator JAR file (`yb-sample-apps.jar`).
+Download the YugabyteDB workload generator JAR file (`yb-sample-apps.jar`) using the following command:
 
 ```sh
 $ wget https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.9/yb-sample-apps.jar?raw=true -O yb-sample-apps.jar
```

Comment:
Delete the prompt?

wget https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.9/yb-sample-apps.jar?raw=true -O yb-sample-apps.jar